parlai/mturk/core/README.md

```diff
@@ -0,0 +1,38 @@
+## Included
+
+`parlai/mturk/core` contains several files and folders that are needed for proper MTurk integration:
+
+### Folders
+
+- **react_server/**: contains the build files for running a task that uses a react frontend. This folder gets cloned, built, and then pushed to the hosting server (usually heroku).
+- **scripts/**: contain various helpful review and cleanup scripts to manage tasks from the command line (especially after mistakes or major failures).
+- **server_legacy/**: contains the build files for running a task that uses the old templated HTML frontend, and is no longer maintained. This folder gets cloned, built, and then pushed to the hosting server (usually heroku).
```

Comment:
quick question - when is it _not_ heroku?

Reply:
Launching tasks locally doesn't use heroku, but that is a very particular case